src/Sylius/Bundle/ChannelBundle/Resources/config/seriailzer/Model.Channel.yml

```diff
@@ -0,0 +1,38 @@
+Sylius\Component\Channel\Model\Channel:
+    exclusion_policy: ALL
+    xml_root_name: channel
+    properties:
```

Comment:
@loevstroem I think we should remove the type for now, I had some ideas, but let's keep it for later iteration.